lib/block-supports/duotone.php

```diff
@@ -421,143 +421,10 @@ function gutenberg_register_duotone_support( $block_type ) {
 	}
 }
 
-/**
- * Render out the duotone stylesheet and SVG.
- *
- * @param  string $block_content Rendered block content.
- * @param  array  $block         Block object.
- * @return string                Filtered block content.
- */
-function gutenberg_render_duotone_support( $block_content, $block ) {
```

Comment:
For those reviewing, this function had modifications and was moved to the `WP_Duotone_Gutenberg` class. It may be a little easier to see the modifications in [this diff with an earlier commit](https://github.com/WordPress/gutenberg/compare/trunk...737b9cb7ee06ebe44a09748abaa8a1d6062916c7?w=1).